x/concentrated-liquidity/incentives_test.go

```diff
@@ -2378,8 +2394,7 @@ func (s *KeeperTestSuite) TestCreateIncentive() {
 					incentiveRecordOne.IncentiveRecordBody.RemainingAmount.Ceil().RoundInt(),
 				),
 			),
-			recordToSet:       incentiveRecordOne,
-			authorizedUptimes: types.SupportedUptimes,
```

Comment:
Note to reviewer: removing cases where authorizedUptimes are not strictly necessary so that we can fall back on a more robust strategy for testing them in #5324